messybrainz/db/testing.py

```diff
@@ -64,6 +65,10 @@ def drop_tables(self):
         db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'drop_tables.sql'))
 
 
+    def drop_db(self):
+        db.run_sql_script_without_transaction(os.path.join(ADMIN_SQL_DIR, 'drop_db.sql'))
```

Comment:
(I thought I wrote this comment, but I don't see it any more)
In AB and LB we create the database once before we run the tests, and then the test setup deletes and recreates the tables. This speeds up the runtime of the tests a bit.
 - a thought - postgres will refuse to drop a database if there are active connections to it. Does sqlalchemy keep any active connections around that might cause a problem with this?

Reply:
This seems like out of scope for this PR, i just did the simplest thing to fix it for now, I've opened a ticket and will fix this in a different PR: https://tickets.metabrainz.org/browse/LB-455